packages/block-editor/README.md

```diff
@@ -881,6 +881,10 @@ _Returns_
 
 -   `?WPBlockBindingsUtils`: Object containing the block bindings utils.
 
+_Changelog_
+
+`6.7.0` Introduced in WordPress core.
+
```

Comment:
I followed what is done in other functions like this one: https://github.com/WordPress/gutenberg/blob/0e17bd28ac8adf8612b9daa2b1d30e1961b7c58d/packages/element/README.md#hydrateroot